src/Reddit.NET/Models/Listings.cs

```diff
@@ -58,17 +58,17 @@ public PostContainer GetByNames(string names)
         /// <param name="listingsGetCommentsInput">A valid ListingsGetCommentsInput instance</param>
         /// <param name="subreddit">The subreddit with the article</param>
         /// <returns>A post and comments tree.</returns>
-        public List<(PostContainer, CommentContainer)> GetComments(string article, ListingsGetCommentsInput listingsGetCommentsInput, string subreddit = null)
```

Comment:
I see no reason for this method to return a list, a single `CommentContainer` seems to be sufficient

Reply:
Thanks for doing this!  Code looks perfect.

Merged into 1.2.